cartridge/failover.lua

```diff
@@ -43,7 +43,17 @@ local ApplyConfigError = errors.new_class('ApplyConfigError')
 local ValidateConfigError = errors.new_class('ValidateConfigError')
 local StateProviderError = errors.new_class('StateProviderError')
 
-vars:new('membership_notification', membership.subscribe())
+vars:new('membership_notification', fiber.channel(1))
+fiber.new(function()
```

Comment:
I suggest you to save this fiber into vars. I think it's not fine to create dangling fiber.